lib/commander/help_formatters/terminal/help.erb

```diff
@@ -6,10 +9,11 @@
 
     <%= Commander::HelpFormatter.indent 4, program(:description) %>
 
-  <%= $terminal.color "COMMANDS", :bold %>:
+  <%= "%s: %s" % [$terminal.color("COMMANDS", :bold), default_command.nil? ? '' : '(* default)'] %>
```

Comment:
@ggilder You're right. I also think direct use of "(default)" is more clear, but * would be more readable. You can see in the real example below.

### With *

<details>
<summary>with * would be readable</summary>

```
  fastlane

  CLI for 'fastlane' - The easiest way to automate beta deployments and releases for your iOS and
Android apps

        Run using `fastlane [platform] [lane_name]`
        To pass values to the lanes use `fastlane [platform] [lane_name] key:value key2:value2`

  Commands: (* default)
        action                 Shows more information for a specific command
        actions                Lists all available fastlane actions
        add_plugin             Add a new plugin to your fastlane setup
        docs                   Generate a markdown based documentation based on the Fastfile
        enable_auto_complete   Enable tab auto completion
        env                    Print your fastlane environment, use this when you submit an issue on
GitHub
        help                   Display global or [command] help documentation
        init                   Helps you with your initial fastlane setup
        install_plugins        Install all plugins for this project
        lanes                  Lists all available lanes and shows their description
        list                   Lists all available lanes without description
        new_action             Create a new custom action for fastlane.
        new_plugin             Create a new plugin that can be used with fastlane
        run                    Run a fastlane one-off action without a full lane
        search_plugins         Search for plugins, search query is optional
        trigger              * Run a specific lane. Pass the lane name and optionally the platform
first.
        update_fastlane        Update fastlane to the latest release
        update_plugins         Update all plugin dependencies

  Global Options:
    --verbose
    --capture_output     Captures the output of the current run, and generates a markdown issue
template
    --troubleshoot       Enables extended verbose mode. Use with caution, as this even includes ALL
sensitive data. Cannot be used on CI.
    --env STRING[,STRING2] Add environment(s) to use with `dotenv`
    -h, --help           Display help documentation
    -v, --version        Display version information

  Options for trigger:
    --env STRING[,STRING2] Add environment(s) to use with `dotenv`

```

</details>

### With (default)

<details>
<summary>with (default) to indicate default_command directly</summary>

```
  fastlane

  CLI for 'fastlane' - The easiest way to automate beta deployments and releases for your iOS and
Android apps

        Run using `fastlane [platform] [lane_name]`
        To pass values to the lanes use `fastlane [platform] [lane_name] key:value key2:value2`

  Commands:
        action               Shows more information for a specific command
        actions              Lists all available fastlane actions
        add_plugin           Add a new plugin to your fastlane setup
        docs                 Generate a markdown based documentation based on the Fastfile
        enable_auto_complete Enable tab auto completion
        env                  Print your fastlane environment, use this when you submit an issue on
GitHub
        help                 Display global or [command] help documentation
        init                 Helps you with your initial fastlane setup
        install_plugins      Install all plugins for this project
        lanes                Lists all available lanes and shows their description
        list                 Lists all available lanes without description
        new_action           Create a new custom action for fastlane.
        new_plugin           Create a new plugin that can be used with fastlane
        run                  Run a fastlane one-off action without a full lane
        search_plugins       Search for plugins, search query is optional
        trigger (default)    Run a specific lane. Pass the lane name and optionally the platform
first.
        update_fastlane      Update fastlane to the latest release
        update_plugins       Update all plugin dependencies

  Global Options:
    --verbose
    --capture_output     Captures the output of the current run, and generates a markdown issue
template
    --troubleshoot       Enables extended verbose mode. Use with caution, as this even includes ALL
sensitive data. Cannot be used on CI.
    --env STRING[,STRING2] Add environment(s) to use with `dotenv`
    -h, --help           Display help documentation
    -v, --version        Display version information

  Options for trigger:
    --env STRING[,STRING2] Add environment(s) to use with `dotenv`

```

</details>

<details>
<summary>with (default) after command can be too long</summary>

```
  Commands:
        action                          Shows more information for a specific command
        actions                         Lists all available fastlane actions
        add_plugin                      Add a new plugin to your fastlane setup
        docs                            Generate a markdown based documentation based on the Fastfile
        enable_auto_complete (default)  Enable tab auto completion
```

</details>